app/models/customer.py

```diff
@@ -0,0 +1,21 @@
+from app import db
+from datetime import datetime
+
+class Customer(db.Model):
+    __tablename__ = "customers"
+    customer_id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String)
+    postal_code = db.Column(db.Integer)
+    phone = db.Column(db.String)
+    registered_at = db.Column(db.DateTime, default=datetime.utcnow())
+    videos_checked_out_count = db.Column(db.Integer, default=0)
+
+    def to_json(self):
+        return {
+            "id": self.customer_id,
+            "name": self.name,
+            "postal_code": self.postal_code,
+            "phone": self.phone,
+            "registered_at": self.registered_at,
+            "videos_checked_out_count": self.videos_checked_out_count
```

Comment:
Once you have the relationship the `videos_checked_out_count` is just the length of the `rentals` that haven't been checked in:

```suggestion
            "videos_checked_out_count": len(rental for rental in self.rentals if not rental.check_in_date)
```